koshigoe/brick_ftp
# Change domain of API endpoint

## What did you implement:

Change domain from `brickftp.com` to `files.com`.

> We launched BrickFTP 10 years ago to keep the world’s files secure, accessible, and organized in the cloud. When we launched, FTP was one of the main protocols in use and the first that we chose to support.
> 
> Since then, BrickFTP has grown to support many more protocols and a suite of enterprise class features to suit a wide variety of file logistics needs. We’ve focused on supporting both legacy and modern protocols and kept a steady eye on the future of global information sharing. BrickFTP has evolved to become a complete solution for all of your File Storage and Transfer needs, and it’s time that our name and branding reflect that.
> 
> So, I am proud to announce that beginning today, BrickFTP has a new name: Files.com.
> 
> To be clear, this is still the same product and team that you’ve come to love! You will continue to be supported by the same outstanding team and nothing will change about pricing or support. You also will have all the same features, plus some exciting new features that we are excited to launch soon.
> 
> Within the next week or two, you will notice that your website will start showing “files.com” in the URL bar, as well as a Files.com logo and color scheme (provided you haven’t customized your logo or colors). Your IP addresses will remain the same.
> 
> As always, if you have any questions, you can chat, email, or call us. Thanks again for being a BrickFTP / Files.com customer, and we look forward to continuing to serve all your file logistics needs.
> 
> https://brickftp.com/2019/05/08/brickftp-new-name-files-com/

## How did you implement it:

- change domain from `brickftp.com` to `files.com`

## How can we verify it:

```
$ bin/console
> api_client = BrickFTP::RESTfulAPI::Client.new(ENV['BRICK_FTP_SUBDOMAIN'], ENV['BRICK_FTP_API_KEY'])
> BrickFTP::RESTfulAPI::ListUsers.new(api_client).call
```

_**NOTE: Check SSL cert of new custom domain.**_


## Todos:


***Is this ready for review?:*** YES
***Is it a breaking change?:*** NO
